wallet-new/cardano-sl-wallet-new.cabal

```diff
@@ -171,6 +176,7 @@ library
                      , network-transport
                      , optparse-applicative
                      , QuickCheck
+                     , mwc-random
```

Comment:
Revisit if the choice of `mwc-random` is the best or if we do need to enforce other and more important properties like privacy, in which case we should just use the `MonadRandom IO` that `Foundation` provides.